src/server.ts

```diff
@@ -10,6 +10,7 @@ import { production, development, testing } from './db/config';
 import session from 'express-session';
 import router from './routes/index';
 require('./associations/associations');
+require("./associations/associations")
```

Comment:
Why duplicates?

Reply:
> Why duplicates?

that was a mistake, i'm going to remove one